### PR TITLE
build: Updated dependencies to latest versions

### LINKED ISSUE #429: at_chops - upgrade dependencies and modify example
### Describe the bug

* at_chops package has outdated dependencies
* at_chops example has deprecated methods

### Steps to reproduce

* https://pub.dev/packages/at_chops/score

### Expected behavior

* at_chops pubspec should have latest date dependencies
* at_chops example should use new methods in place of deprecated methods
* other packages using at_chops should work fine with upgraded dependencies

### Screenshots

_No response_

### Smartphones

_No response_

### Were you using an atApplication when the bug was found?

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by purnimavenkatasubbu**
**- What I did**

- upgrade dependencies for at_chops and at_onboarding_cli to latest version
- updated the changelog to reflect all the upgrades

**- How I did it**

**- How to verify it**
All the tests should pass
**- Description for the changelog**

### CODE REVIEW

File: packages/at_chops/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## 1.0.7
+- build[deps]: Upgraded the following packages:
+    - at_commons to v3.0.58
```
Comment (sitaram-kalluri) [inline packages/at_chops/CHANGELOG.md]: Please update the version of at_commons to 4.0.0

File: packages/at_chops/pubspec.yaml [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-  at_utils: ^3.0.15
+  elliptic: ^0.3.10
+  pointycastle: ^3.7.3
+  at_commons: ^4.0.0
+  at_utils: ^3.0.16
 
 dev_dependencies:
-  lints: ^2.0.0
+  lints: ^3.0.0
   test: ^1.16.0
```
Comment (sitaram-kalluri) [inline packages/at_chops/pubspec.yaml]: Please update the test package dependency also to a latest version

**Review by sitaram-kalluri — CHANGES_REQUESTED** [review]

File: packages/at_chops/pubspec.yaml (hunk 2 above)
Comment (purnimavenkatasubbu) [reply packages/at_chops/pubspec.yaml]: updated

File: packages/at_chops/CHANGELOG.md (hunk 1 above)
Comment (purnimavenkatasubbu) [reply packages/at_chops/CHANGELOG.md]: updated

**Review by sitaram-kalluri — APPROVED** [review]

**Comment by purnimavenkatasubbu** [comment]
Closing this PR as there's already a branch with at_commons upgrade. I'll use the same branch and raise a [PR](uptake_commons_into_atchops)

**Pull request closed without merging**